petbuddy-homepage/src/styles/global.css

```
@@ -12,3 +13,32 @@
   align-items: center;
   width: auto;
 }
+
+/* styles navigation bar */
+.header {
+  display: flex;
+  height: 80;
+  width: auto;
+  object-fit: cover;
+  background-color: #f5f5f5;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 50px;
+  justify-content: space-between;
+}
+
+.boxNavigation a,
```

```suggestion
.box-navigation a,
```
The previous is a common css naming convention.

---

Change all class attributes

---

Change in [5a97ec9](https://github.com/OwlbyTech/PetBuddy-homepage/pull/3/commits/5a97ec9b4daee6b84c81e171eaf0dd18eadb1853)